script.module.resolveurl/lib/resolveurl/plugins/doodstream.py

```diff
@@ -32,7 +32,7 @@ class DoodStreamResolver(ResolveUrl):
     pattern = r'(?://|\.)(dood(?:stream)?\.(?:com?|watch|to|s[ho]|cx|la|w[sf]|pm|re))/(?:d|e)/([0-9a-zA-Z]+)'
 
     def get_media_url(self, host, media_id):
-        if host.endswith('.cx'):
+        if host.endswith('.wf'):
```

Comment:
you need to add checking for `.wf` and not just replace `.cx` as that is still required
`if any(host.endswith(x) for x in ['.cx', '.wf']):` will do the job